cmd/version.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+var Version = "dev"
```

Comment:
(nit) if this var being exported for any reason? 
If it's not, I would prefer it to be declared inside `newVersionCmd` to avoid collisions with other variables in the same package.

Reply:
+1

Reply:
I thought I needed to export the variable in order to set the value form ldflags but it is not needed. I just changed it